src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -1606,11 +1632,69 @@ impl<T: AstInfo> AstDisplay for AlterClusterStatement<T> {
 }
 impl_display_t!(AlterClusterStatement);
 
+/// `ALTER REPLICA SET .. SET ...`
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct AlterReplicaSetStatement<T: AstInfo> {
+    /// The `IF EXISTS` option.
+    pub if_exists: bool,
+    /// Name of the altered cluster.
+    pub name: QualifiedClusterItem,
+    /// The action.
+    pub action: AlterReplicaSetAction<T>,
+}
+
+impl<T: AstInfo> AstDisplay for AlterReplicaSetStatement<T> {
+    fn fmt<W: fmt::Write>(&self, f: &mut AstFormatter<W>) {
+        f.write_str("ALTER REPLICA SET ");
+        if self.if_exists {
+            f.write_str("IF EXISTS ");
+        }
+        f.write_node(&self.name);
+        f.write_str(" ");
+        match &self.action {
+            AlterReplicaSetAction::SetOptions(options) => {
+                f.write_str("SET (");
+                f.write_node(&display::comma_separated(options));
+                f.write_str(")");
+            }
+            AlterReplicaSetAction::ResetOptions(options) => {
+                f.write_str("RESET (");
+                f.write_node(&display::comma_separated(options));
+                f.write_str(")");
+            }
+        }
+    }
+}
```

Comment:
According to coverage data, this method is not covered by any end-to-end test. Would it make sense to add a test for this or is this more about debugging output?